model/version.go

```diff
@@ -465,7 +465,7 @@ type MainlineCommitVersionOptions struct {
 func GetMainlineCommitVersionsWithOptions(projectId string, opts MainlineCommitVersionOptions) ([]Version, error) {
 	invalidRequesters, _ := utility.StringSliceSymmetricDifference(opts.Requesters, evergreen.SystemVersionRequesterTypes)
 	if len(invalidRequesters) > 0 {
-		return nil, errors.Errorf("invalid requesters: %v", invalidRequesters)
+		return nil, errors.Errorf("invalid requesters %s", invalidRequesters)
```

Comment:
`'%s'`

Reply:
Same thing about this being a string slice.